4_system_architecture.md

```diff
@@ -0,0 +1,151 @@
+
+Solar home system picture
+
+:::
+
+A neighborhood of such solar home systems may form another standard conforming microgrid operating as a grid of grids.
+
+::: warning Need picture
+
+Connected neighborhood pictures.
+
+:::
+
+Extending the model further, neighborhoods may be combined into another microgrid as grid of grid of grids and so on.
+
+::: warning Need picture  
+
+Grid of grid picture.
+
+:::
+
+
+It is this arbitrary depth of hierarchy that causes this standard to refrain from using other terms that are sometimes
+used to refer to particular positions in the hierarchy such as nanogrid, picogrid etc.
+
+## Link Properties
+
+The properties of a link include its range of permitted voltages and currents and a communications technology associated with the link.
+This standard defines multiple types of links as subsections of the standard. A device conforms to the standard if it has at least
+on grid port that conforms to one of the types of links defined in the standard.  It may have other electrical ports that do not
+conform to any of the standard links.
+
+
+The DC grid allows to interconnect multiple energy sources and sinks.
+
+Compared to conventional grids, where the energy flow is mostly uni-directional from the energy source to the consumer, the meshed DC grid can operate in both directions. A battery for example can be a current source or a current sink.
+
+In order to explain the grid topology, the overall system is divided into abstract sub-components, as described below. These components don't necessarily need to be separate physical devices, but can be different sub-components in one circuit.
+
+### Power line
+
+Bipolar connection (cable) between exactly two grid ports. Splices are not allowed.
+
+A power line might cover long distances, resulting in a voltage drop caused by the resistance of the cable. The voltage drop has to be considered to allow stable system control.
+
+### DC bus
+
+Connection of two or more electrical circuits such that the electric potential at each connection point can be considered equal. This can be achieved either by connecting each participant very close to each other or by using large copper cross-section such that voltage drop introduced by current flow is negligible.
+
+The cross section of the conductor must be large enough to handle the sum of the maximum current of all connected sources.
+
+### DC grid port
+
+Connection between a DC bus and a power line, featuring
+
+- a circuit breaker or electronic switch to allow disconnection of the port for safety reasons or to optimize energy flows and
+- measurement of port current and bus voltage.
+
+### Voltage converter
+
+A DC/DC converter, necessary to connect DC buses of different voltage levels, e.g. a battery or solar panel to the DC grid.
+
+
+### System overview
+
+The following image gives an example of a possible DC grid layout.
+
+<center>
+
+![Example of a DC grid system layout](./images/grid-topology.png)
```

Comment:
Now we've got this picture twice.

I will translate it to SVG and also make some pictures to better explain bus configuration etc. of previous chapters.